CONTRIBUTING.md

```diff
@@ -0,0 +1,151 @@
+# Contributing
+
+**We appreciate all kinds of help, so thank you!** :clap: :kissing_heart:
+
+You can contribute in many ways to this project.
+
+## Issue reporting
+
+:fire: This is a good point to start, when you find a problem please add it to the [issue tracker](https://github.com/Qiskit/openqasm/issues). Please, use [this template](https://github.com/Qiskit/openqasm/contributing/blob/main/templates/issue.md) to add them.
```

Comment:
We don't have this template and I doubt we need it. Suggest dropping this sentence.

Reply:
Fixed in 4e8bb1e.